File: solutions/2025-dec/16/consonant_count.js
function hasConsonantCount(text, target) {
    const vowels = ["a", "e", "i", "o", "u"];
    let numConsonants = 0;
    for (const c of text) {
        if (!/^[a-z]+$/i.test(c)) continue;
        if (!vowels.includes(c)) numConsonants++;
    }
    return numConsonants == target;
}

File: solutions/2025-dec/16/consonant_count.py
def has_consonant_count(text, target):
    vowels = set[str](["a", "e", "i", "o", "u"])
    numConsonants = 0
    for c in text.lower():
        if not c.isalpha():
            continue
        if c not in vowels:
            numConsonants += 1
    return numConsonants == target
